Packaging/OpenDingux/build.sh

```diff
@@ -79,14 +79,33 @@ build() {
 	mkdir -p ../../build
 	cd ../../build
 	rm -f CMakeCache.txt
+
+	local -a defs=(-DDINGUX=ON -DBINARY_RELEASE=ON)
 	if [[ "$TARGET" == "rg350" ]]; then
-		TARGET_DEFINES="-DNONET=ON"
+		defs+=(-DNONET=1)
```

Comment:
Does this affect documentation, or will -DNONET=ON continue to work?

Reply:
There is no reason for this change anymore, it was only needed in an earlier version of my code. TRUE, ON, YES, 1, all work with CMake.

Reply:
Ok feel free to remove the changes if the nolonger gives any benifit 🙂

Reply:
Done